src/LieTheory/WeylGroup.jl

```diff
@@ -753,6 +753,42 @@ function reflection(beta::RootSpaceElem)
   return W([list_of_indices; current_index; reverse(list_of_indices)])
 end
 
+###############################################################################
+# G-set functionality
+
+function gset(G::WeylGroup; closed::Bool=false, check::Bool=true)
+  return GSetByElements(G, *, roots(root_system(G)); closed=closed, check=check)
```

Comment:
I don't see a good reason to accept the two kwargs here. e.g. `gset(::PermGroup)` doesn't do that as well.
```suggestion
function gset(G::WeylGroup)
  return GSetByElements(G, *, roots(root_system(G)); closed=is_finite(G), check=false)
```

I think I would find it even more intuitive if it would be routed through https://github.com/oscar-system/Oscar.jl/blob/23dacb3a4e2caf6541ec46aa52912243f3f5a15f/src/Groups/gsets.jl#L171 and then we have suitable `gset_by_type` methods for `::Type{RootSpaceElem}` and `::Type{WeightLatticeElem}` that then input the action function `*`

Reply:
Fine by me